cyrasterize/shader.pyx

```diff
@@ -46,13 +47,14 @@ cdef class ShaderSource:
                 str(self), error))
 
         log.info('Compiled a {} ({}) shader'.format(str(self), self.shader_type))
+        self._compiled = True
 
     def __dealloc__(self):
         # if self.shader != -1:
         log.debug('TODO Maybe we have to deallocate the shader here?')
 
     cpdef bool is_compiled(self):
-        return self.uid != -1
+        return self._compiled
```

Comment:
Simple change to fix compiler warning as `self.uid` is `unsigned` so it can never be `-1`.